utils/argvProcessor.mjs

```diff
@@ -0,0 +1,83 @@
+import _ from 'underscore';
+
+export default class ArgvProcessor {
+    constructor (args) {
+        this.args = args;
+        this.actionShortCuts = {
+                'r': 'reverse',
+                't': 'transform',
+                'o': 'outputFile',
+                'cf': 'convertFromFile',
+                'ct': 'convertToFile',
+                'cb': 'cssBundler',
+        };
+        this.helpMessage = 'Help message';
+    }
+
+    run () {
+        this.hasHelper() ? this.showHelp() : this.processArguments();
```

Comment:
Use ordinary if condition syntax, please

If develpoer will add new methods it will be more than 1 line in diff.